### PR TITLE
updated readme & started on routes

### PR DESCRIPTION
**Pull request by gorakong**

### CODE REVIEW

**Comment by gorakong** [comment]
note i havent actually included the Controllers file

**Comment by nvodoor** [comment]
Approved. Do not have write access so I can't merge it in. However, merge is approved.

**Review by nvodoor — APPROVED** [review]

**Comment by sveta0112** [comment]
@gorakong Looks good

**Pull request merged**
